rust/README.md

```diff
@@ -40,4 +48,17 @@ TODO
 
 ### Code Generation
 
-TODO
+If you do changes to either
+* `ast-generator`, or
+* `schema/*.py`
+
+you'll need to regenerate code. You can do so running
+```bash
```

Comment:
Mega nitpick, but I think it's cleaner to use `sh` for things that works in any shell and not just Bash.
```suggestion
```sh
```